pysensibo/__init__.py

```diff
@@ -205,7 +205,7 @@ async def async_set_ac_state_property(
     async def _get(self, path: str, params: dict[str, Any]) -> dict[str, Any]:
         """Make GET api call to Sensibo api."""
         async with self._session.get(path, params=params, timeout=self.timeout) as resp:
-            if resp.status == 401:
+            if resp.status in [401, 403]:
```

Comment:
It would have been nice to make these a constant HTTP_AUTH_FAILED_STATUS_CODES = {401, 403}